layers/src/main/scala/geotrellis/layers/hadoop/HadoopAttributeStore.scala

```diff
@@ -32,11 +32,11 @@ import java.io.PrintWriter
 
 class HadoopAttributeStore(
   val rootPathString: String,
-  val conf: Configuration
+  private val serConf: SerializableConfiguration // This needs to be serializable
```

Comment:
Not sure I see what `private` is for here: if it is private, don't make it a val?

Reply:
Actually, I'm not sure either constructor has much use for being a `val`

Reply:
I just made it `private` so that the user doesn't access the `SerializableConfiguration` which is something we're trying to hide. Though you're right, they probably don't need to be `val`.